### ISSUE TITLE
Godot crashes after latest Nvidia driver update

### ISSUE THREAD

**Issue by Skyvastern** [godotengine/godot#77806]
### Godot version

4.0.3.stable

### System information

Windows 10 64-bit, 8GB RAM, Nvidia GTX 1650 (Laptop)

### Issue description

After updating Nvidia driver to the latest v535.98 release, I'm no longer able to open Godot project that uses -
- Forward+
- Mobile

The Compatibility mode works fine.

### Steps to reproduce

Using the Nvidia's v535.98, and creating/opening a project that uses either Forward+ or Mobile, gives the following error -

![image](https://github.com/godotengine/godot/assets/52847458/12906d4e-d3c0-41f3-ab15-54254b1ff607)

Even after a reboot the error still shows up.

### Minimal reproduction project

N/A

**Comment by Calinou** [comment]
Can you run the Godot editor from the [command line](https://docs.godotengine.org/en/stable/tutorials/editor/command_line_tutorial.html) with the `--verbose` argument and paste the output there?

**Comment by Skyvastern** [comment]
> Can you run the Godot editor from the [command line](https://docs.godotengine.org/en/stable/tutorials/editor/command_line_tutorial.html) with the `--verbose` argument and paste the output here?
> 
> Also, what integrated graphics do you have? If you have AMD integrated graphics, see #57708.

Going through #57708 I was able to fix the issue, I had to update the AMD Radeon Graphics driver, and the error went away 😀

Also just an update which I wasn't able to add to this thread, that even when I rolled back to previous version of Nvidia's driver, I was somehow still getting that error, which I wasn't getting yesterday... weird 🤔

But anyways, it got fixed after updating AMD drivers. And yes I'm using the latest Nvidia drivers right now.

Thanks a lot! 😊
